kalite/distributed/api_views.py

```diff
@@ -82,9 +84,25 @@ def compute_total_points(user):
         return None
     else:
         return (VideoLog.get_points_for_user(user) +
-           ExerciseLog.get_points_for_user(user) +
-           StoreTransactionLog.get_points_for_user(user) +
-           ContentLog.get_points_for_user(user))
+                ExerciseLog.get_points_for_user(user) +
```

Comment:
Sorry, I have a tendency to make comments for the sake of it. If there is no meaning evident (or I am speaking in doge speak), I am just being approving.